src/main/webapp/css/home_style.css

```diff
@@ -0,0 +1,13 @@
+.heading {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  height: 45vh;
+}
+
+h1 {
+  text-align: center;
```

Comment:
This does nothing due to the way you do align-items: center, which forces the width of the h1 to be auto, which means centering doesn't make sense.